fix for sdk linter retuning false warning (#3112)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased

> - Fixed an issue in *lint* where it returned warnings for some unrelated files.

Reviewer: what unrelated files?

## Unreleased
* Added a check to **is_docker_image_latest_tag** to only fail the validation on non-latest image tag when the current tag is older than 3 days.
* Fixed an issue where **upload** would not properly show the installed version in the UI.

> * Fixed an issue in *lint* where it returned a warning regarding usage of `demisto.results` function in entities that have to use it and can't use `CommandResults` instead.

Reviewer: ```suggestion
* Fixed an issue where **lint** falsely warned of using `demisto.results`.
```
